### ISSUE TITLE
What are the dependences of repo?

### ISSUE THREAD

**Issue by Kracozebr** [SMohammadi89/PointView-GCN#6]
Hi, thanks for sharing your code, I want to work with your repo, but can't find any dependences. Could your share them, please?

**Comment by SMohammadi89** [comment]
Hi,


_openmp_mutex | 5.1 | 5.1
absl-py | 0.12.0 | 1.3.0
addict | 2.4.0 |  
aiocontextvars | 0.2.2 |  
alsa-lib | 1.2.3 |  
anyio | 3.3.0 | 3.5.0
argon2-cffi | 21.1.0 | 21.3.0
async_generator | 1.10 | 1.10
attrs | 21.2.0 | 22.1.0
babel | 2.9.1 | 2.11.0
backcall | 0.2.0 | 0.2.0
backports | 1.0 | 1.1
backports.functools_lru_cache | 1.6.4 | 1.6.4
binvox | 0.1.5 |  
blas | 1.0 | 1.0
bleach | 4.1.0 | 4.1.0
brotlipy | 0.7.0 | 0.7.0
bzip2 | 1.0.8 | 1.0.8
ca-certificates | 2019.11.28 | 2023.01.10
cairo | 1.16.0 | 1.16.0
certifi | 2021.5.30 | 2022.12.7
cffi | 1.14.6 | 1.15.1
chardet | 4.0.0 | 4.0.0
charset-normalizer | 2.0.0 | 2.0.4
click | 8.0.1 | 8.0.4
contextvars | 2.4 | 2.4
cryptography | 3.4.7 | 38.0.4
cubic-feature-sampling | 1.1.0 |  
cudatoolkit | 10.2.89 | 11.3.1
curl | 7.71.1 | 7.87.0
cycler | 0.10.0 | 0.11.0
dataclasses | 0.8 | 0.8
dbus | 1.13.18 | 1.13.18
decorator | 5.0.9 | 5.1.1
defusedxml | 0.7.1 | 0.7.1
deprecated | 1.2.12 | 1.2.13
deprecation | 2.1.0 | 2.1.0
docker-pycreds | 0.4.0 | 0.4.0
double-conversion | 3.1.5 | 3.1.5
eigen | 3.3.9 | 3.3.7
emd | 0.0.0 |  
entrypoints | 0.3 | 0.4
expansion-penalty | 0.0.0 |  
expat | 2.4.1 | 2.4.9
ffmpeg | 4.3.2 | 4.2.2
fontconfig | 2.13.1 | 2.14.1
freetype | 2.10.4 | 2.12.1
future | 0.18.2 | 0.18.2
gettext | 0.19.8.1 | 0.21.0
gitdb | 4.0.9 | 4.0.7
gitpython | 3.1.18 | 3.1.30
gl2ps | 1.4.2 | 1.4.2
glew | 2.1.0 | 2.1.0
glib | 2.68.3 | 2.69.1
glib-tools | 2.68.3 |  
gmp | 6.2.1 | 6.2.1
gnutls | 3.6.13 | 3.6.15
graphite2 | 1.3.13 | 1.3.14
gridding | 2.1.0 |  
grpcio | 1.38.0 | 1.42.0
gst-plugins-base | 1.18.4 | 1.18.5
gstreamer | 1.18.4 | 1.18.5
h5py | 2.10.0 | 3.7.0
harfbuzz | 2.8.2 | 4.3.0
hdf4 | 4.2.13 | 4.2.13
hdf5 | 1.10.6 | 1.12.1
icu | 68.1 | 68.1
idna | 3.1 | 3.4
immutables | 0.16 | 0.16
importlib-metadata | 4.3.1 | 4.11.3
intel-openmp | 2022.0.1 | 2022.1.0
ipykernel | 5.5.5 | 6.19.2
ipython | 7.16.1 | 8.8.0
ipython-genutils | 0.2.0 |  
ipython_genutils | 0.2.0 | 0.2.0
ipywidgets | 7.6.4 | 7.6.5
jasper | 1.900.1 | 2.0.14
jbig | 2.1 | 2.1
jedi | 0.18.0 | 0.18.1
jinja2 | 3.0.1 | 3.1.2
joblib | 1.0.1 | 1.1.1
jpeg | 9d | 9e
json5 | 0.9.6 | 0.9.6
jsoncpp | 1.8.4 | 1.9.4
jsonpatch | 1.24 | 1.32
jsonpointer | 2.0 | 2.1
jsonschema | 3.2.0 | 4.16.0
jupyter | 1.0.0 | 1.0.0
jupyter-client | 7.0.2 |  
jupyter-core | 4.7.1 |  
jupyter-packaging | 0.10.4 | 0.12.0
jupyter-server | 1.10.2 |  
jupyter_client | 7.1.0 | 7.4.8
jupyter_console | 5.2.0 | 6.4.4
jupyter_core | 4.8.1 | 5.1.1
jupyter_server | 1.4.1 | 1.23.4
jupyterlab | 3.1.10 | 3.5.2
jupyterlab-pygments | 0.1.2 |  
jupyterlab-server | 2.8.1 |  
jupyterlab-widgets | 1.0.1 |  
jupyterlab_server | 2.8.2 | 2.16.5
jupyterlab_widgets | 1.0.2 | 1.0.0
kiwisolver | 1.3.1 | 1.4.4
krb5 | 1.19.2 | 1.19.4
lame | 3.100 | 3.100
lcms2 | 2.12 | 2.12
ld_impl_linux-64 | 2.36.1 | 2.38
lerc | 2.2.1 | 3.0
libblas | 3.9.0 |  
libcblas | 3.9.0 |  
libclang | 11.1.0 | 14.0.6
libcurl | 7.71.1 | 7.87.0
libdeflate | 1.7 | 1.8
libedit | 3.1.20191231 | 3.1.20221030
libevent | 2.1.10 | 2.1.12
libffi | 3.3 | 3.4.2
libgcc-ng | 9.3.0 | 11.2.0
libgfortran-ng | 7.5.0 | 11.2.0
libgfortran4 | 7.5.0 | 7.5.0
libglib | 2.68.3 |  
libglu | 9.0.0 | 9.0.0
libgomp | 9.3.0 | 11.2.0
libiconv | 1.16 | 1.16
liblapack | 3.9.0 |  
liblapacke | 3.9.0 |  
libllvm11 | 11.1.0 | 11.1.0
libnetcdf | 4.7.4 | 4.8.1
libogg | 1.3.4 | 1.3.5
libopencv | 4.5.2 | 4.6.0
libopus | 1.3.1 | 1.3.1
libpng | 1.6.37 | 1.6.37
libpq | 13.3 | 12.9
libprotobuf | 3.16.0 | 3.20.1
libsodium | 1.0.18 | 1.0.18
libssh2 | 1.9.0 | 1.10.0
libstdcxx-ng | 9.3.0 | 11.2.0
libtheora | 1.1.1 | 1.1.1
libtiff | 4.3.0 | 4.5.0
libuuid | 2.32.1 | 1.41.5
libuv | 1.40.0 | 1.40.0
libvorbis | 1.3.7 | 1.3.7
libwebp-base | 1.2.0 | 1.2.4
libxcb | 1.14 | 1.15
libxkbcommon | 1.0.3 | 1.0.1
libxml2 | 2.9.12 | 2.9.14
loguru | 0.5.3 | 0.5.3
lxml | 4.7.1 | 4.9.1
lz4-c | 1.9.3 | 1.9.4
markdown | 3.3.4 | 3.4.1
markupsafe | 2.0.1 | 2.1.1
matplotlib | 3.3.4 | 3.6.2
matplotlib-base | 3.3.4 | 3.6.2
mds | 0.0.0 |  
mistune | 0.8.4 | 2.0.4
mkl | 2020.2 | 2022.1.0
mkl-service | 2.3.0 | 2.4.0
mkl_fft | 1.3.0 | 1.3.1
mkl_random | 1.1.1 | 1.2.2
multicore-tsne | 0.1_d4ff4aab |  
munch | 2.5.0 | 2.5.0
mysql-common | 8.0.25 |  
mysql-libs | 8.0.25 |  
nbclassic | 0.3.1 | 0.4.8
nbclient | 0.5.4 | 0.5.13
nbconvert | 6.0.7 | 6.5.4
nbformat | 5.1.3 | 5.7.0
ncurses | 6.2 | 6.4
nest-asyncio | 1.5.1 | 1.5.6
nettle | 3.6 | 3.7.3
ninja | 1.10.2 | 1.10.2
notebook | 6.4.3 | 6.5.2
nspr | 4.30 | 4.33
nss | 3.67 | 3.74
numpy | 1.19.2 | 1.23.5
numpy-base | 1.19.2 | 1.23.5
olefile | 0.46 | 0.46
open3d | 0.13.0 |  
opencv | 4.5.2 | 4.6.0
openh264 | 2.1.1 | 2.1.1
openjpeg | 2.3.0 | 2.4.0
openssl | 1.1.1k | 1.1.1s
opt_einsum | 3.3.0 | 3.3.0
packaging | 21.0 | 22.0
pandas | 1.1.5 | 1.5.2
pandoc | 2.18 | 2.12
pandocfilters | 1.4.3 | 1.5.0
parso | 0.8.2 | 0.8.3
pathtools | 0.1.2 | 0.1.2
pcre | 8.45 | 8.45
pexpect | 4.8.0 | 4.8.0
pickleshare | 0.7.5 | 0.7.5
pillow | 8.3.1 | 9.3.0
pip | 21.2.2 | 22.3.1
pixman | 0.40.0 | 0.40.0
plumed | 2.8.0 |  
plyfile | 0.7.4 |  
proj | 7.1.1 | 8.2.1
prometheus-client | 0.11.0 |  
prometheus_client | 0.12.0 | 0.14.1
promise | 2.3 | 2.3
prompt-toolkit | 3.0.20 | 3.0.36
prompt_toolkit | 1.0.15 | 3.0.36
protobuf | 3.17.1 | 3.20.1
psutil | 5.8.0 | 5.9.0
ptyprocess | 0.7.0 | 0.7.0
pugixml | 1.11.4 | 1.11.4
py-opencv | 4.5.2 | 4.6.0
pycparser | 2.20 | 2.21
pygments | 2.10.0 | 2.11.2
pyopenssl | 20.0.1 | 22.0.0
pyparsing | 2.4.7 | 3.0.9
pyqt | 5.12.3 | 5.15.7
pyqt-impl | 5.12.3 |  
pyqt5-sip | 4.19.18 | 12.11.0
pyqtchart | 5.12 | 5.15.7
pyqtwebengine | 5.12.1 | 5.15.7
pyro-api | 0.1.2 |  
pyro-ppl | 1.7.0 |  
pyrsistent | 0.18.0 | 0.18.0
pysocks | 1.7.1 | 1.7.1
python | 3.6.13 | 3.11.0
python-dateutil | 2.8.2 | 2.8.2
python-pptx | 0.6.21 |  
python_abi | 3.6 |  
pytorch | 1.8.1 | 1.12.1
pytorch-metric-learning | 0.9.99 |  
pytz | 2021.1 | 2022.7
pyyaml | 5.4.1 | 6.0
pyzmq | 19.0.2 | 23.2.0
qt | 5.12.9 | 5.15.9
qtconsole | 5.2.1 | 5.4.0
qtpy | 1.11.3 | 2.2.0
readline | 8.1 | 8.2
requests | 2.26.0 | 2.28.1
requests-unixsocket | 0.2.0 |  
scikit-learn | 0.24.2 | 1.2.0
scipy | 1.5.2 | 1.9.3
seaborn | 0.11.2 | 0.12.2
send2trash | 1.8.0 | 1.8.0
sentry-sdk | 1.5.12 | 1.9.0
setproctitle | 1.2.2 | 1.2.2
setuptools | 52.0.0 | 65.6.3
shortuuid | 1.0.1 |  
simplegeneric | 0.8.1 | 0.8.1
sip | 4.19.8 | 6.6.2
six | 1.16.0 | 1.16.0
smmap | 3.0.5 | 4.0.0
sniffio | 1.2.0 | 1.2.0
sqlite | 3.36.0 | 3.40.1
tbb | 2020.2 | 2021.6.0
tbb-devel | 2020.2 | 2021.6.0
tensorboard | 1.15.0 | 2.10.0
tensorboardx | 1.4 | 2.2
terminado | 0.12.1 | 0.17.1
testpath | 0.5.0 | 0.6.0
threadpoolctl | 2.2.0 | 2.2.0
tk | 8.6.10 | 8.6.12
tomlkit | 0.7.2 | 0.11.1
torchfile | 0.1.0 |  
torchvision | 0.2.1 | 0.13.1
tornado | 6.1 | 6.2
tqdm | 4.59.0 | 4.64.1
traitlets | 4.3.3 | 5.7.1
typing_extensions | 3.10.0.0 | 4.4.0
urllib3 | 1.26.6 | 1.26.14
utfcpp | 3.2.1 | 3.2.1
vedo | 2021.0.5 |  
visdom | 0.1.8.9 |  
vtk | 9.0.1 | 9.0.3
wandb | 0.12.16 |  
wcwidth | 0.2.5 | 0.2.5
webencodings | 0.5.1 | 0.5.1
websocket-client | 0.57.0 | 0.58.0
werkzeug | 2.0.1 | 2.2.2
wheel | 0.37.0 | 0.37.1
widgetsnbextension | 3.5.1 | 4.0.3
wrapt | 1.12.1 | 1.14.1
x264 | 1!161.3030 | 1!157.20191217
xlsxwriter | 3.0.2 | 3.0.3
xorg-kbproto | 1.0.7 |  
xorg-libice | 1.0.10 |  
xorg-libsm | 1.2.3 |  
xorg-libx11 | 1.7.2 |  
xorg-libxext | 1.3.4 |  
xorg-libxrender | 0.9.10 |  
xorg-libxt | 1.2.1 |  
xorg-renderproto | 0.11.1 |  
xorg-xextproto | 7.3.0 |  
xorg-xproto | 7.0.31 | 7.0.31
xz | 5.2.5 | 5.2.10


</body>
</html>

**Comment by Kracozebr** [comment]
I do not see dependencies for pointnet++
